src/components/DatePicker/index.android.js

```diff
@@ -8,87 +8,76 @@ import CONST from '../../CONST';
 import {propTypes, defaultProps} from './datepickerPropTypes';
 import styles from '../../styles/styles';
 
-class DatePicker extends React.Component {
-    constructor(props) {
-        super(props);
-
-        this.state = {
-            isPickerVisible: false,
-        };
-
-        this.showPicker = this.showPicker.bind(this);
-        this.setDate = this.setDate.bind(this);
-    }
+function DatePicker({value, defaultValue, label, placeholder, errorText, containerStyles, disabled, maxDate, minDate, innerRef, onBlur, onInputChange}) {
```

Comment:
I think we could remove `DatePicker.defaultProps = defaultProps;` and put defaults here, for consistency with the new pattern

Reply:
Please don't do that yet, it can create problems with ESLint. We will remove `defaultProps` during TS migration.